content/06.multiple-approaches-required.md

```diff
@@ -1,36 +1,30 @@
 #### Multiple approaches are required 
 
-<!-- aka "Putting it all together" -->
+We have described multiple approaches for maximizing the success of machine learning applications in rare disease research throughout. 
+In practice, it is rarely sufficient to use one of these techniques in isolation. 
+In this section, we will discuss two recent works – one supervised and one unsupervised – in the rare disease domain that draw on multiple concepts covered here.
+Feature-representation-transfer (and therefore representation learning and prior data) and regularization underlie both approaches. 
```

Comment:
```suggestion
These two approaches combine dimension reduction techniques like representation learning with prior knowledge and use transfer learning techniques to perform feature-representation-transfer from large related datasets to smaller datasets.
Furthermore, in these examples feature-representation-transfer utilizes regularization to identify disease-relevant features that are useful for constructing simple but stable models.
```

Reply:
As I re-read the lines following this, I think you also hit the exact points that I was thinking of (just in different words). I am still adding these lines here, but you can decide to keep them or not :)